src/bootstrap-table.js

```diff
@@ -2914,6 +2947,11 @@
         this.updatePagination();
     };
 
+    BootstrapTable.prototype.toggleFullscreen = function () {
+        var bootstraptablediv = $(".bootstrap-table");
+        bootstraptablediv.toggleClass("fullscreen");
```

Comment:
I think here can use `this.$el.closest('.bootstrap-table').toggleClass('fullscreen');` instead, because `$(".bootstrap-table")` will return all tables.